src/calculating.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const calculating = (number, answer) => {
+  const bulls = [];
+  const cows = [];
+
+  const len = answer.toString().length;
+
+  for (let i = 0; i < len; i++) {
+    const pcValue = number.toString().split('')[i];
+    const inputValue = answer.toString().split('')[i];
+
+    if (pcValue === inputValue) {
+      bulls.push(inputValue);
+    } else if (pcValue !== inputValue) {
```

Comment:
If you need to check whether `pcValue` is equal or not equal to `inputValue`, it is better to use the `if...else` construction (not `if...else if`).